userspace/falco/configuration.cpp

```diff
@@ -447,6 +447,11 @@ void falco_configuration::load_yaml(const std::string &config_name) {
 		keep_alive = m_config.get_scalar<bool>("http_output.keep_alive", false);
 		http_output.options["keep_alive"] = keep_alive ? std::string("true") : std::string("false");
 
+		uint32_t max_consecutive_timeouts;
+		max_consecutive_timeouts =
+		        m_config.get_scalar<uint32_t>("http_output.max_consecutive_timeouts", 5);
```

Comment:
Shouldn't `m_max_consecutive_timeouts` be enclosed in a range to avoid having an "almost" infinite loop if someone inputs a high value ?

Maybe also use a type smaller than unit32 to store it.

Reply:
ok i see, so an uint16_t or uint8_t should be better @sgaist ?

Reply:
unit8's max being 255, I think it should already cover most use cases.

If there's a need in the future to have more than that number of retries, it's always possible to update at this time.

Reply:
uint8_t makes sense IMHO!